### DISCUSSION TITLE
Translations not working

### DISCUSSION THREAD

**Discussion by emidiotorre** [directus/directus#10712] [Archive]
Hi,
i am having trouble creating the translations.

After i have created the field with the wizard, i go to the content page, and see this
<img width="876" alt="Schermata 2021-12-27 alle 19 27 02" src="https://user-images.githubusercontent.com/10254639/147497993-595c248b-135f-4482-9bee-72e351184678.png">

but nothing is happening after I click on a language.

I worked on earlier versions, i had trouble starting from v. 9.0.0

### To Reproduce

Just create a fresh install, and add translations, I am using Sqlite3 as a DB.

**Comment by rijkvanzanten** [comment]
Heya! It looks like you haven't setup any fields yet in the translations collection. Go to Settings > Data Model and choose the `<collection>_translations` collection to configure what fields you want to be translatable 👍🏻
